docs/reference/monitoring/configuring-monitoring.asciidoc

```diff
@@ -22,13 +22,37 @@ To learn about monitoring in general, see
 
 . Configure your cluster to collect monitoring data:
 
-.. Verify that the `xpack.monitoring.enabled`, 
-`xpack.monitoring.collection.enabled`, and 
-`xpack.monitoring.elasticsearch.collection.enabled` settings are `true` on each
-node in the cluster. By default `xpack.monitoring.collection.enabled` is disabled
-(`false`), and that overrides `xpack.monitoring.elasticsearch.collection.enabled`,
-which defaults to being enabled (`true`). Both settings can be set dynamically
-at runtime. For more information, see <<monitoring-settings>>.
+.. Verify that the `xpack.monitoring.enabled` setting is `true`, which is its 
+default value, on each node in the cluster. For more information, see 
+<<monitoring-settings>>. 
+
+.. Set the `xpack.monitoring.collection.enabled` and 
+`xpack.monitoring.elasticsearch.collection.enabled` settings to `true` on each
+node in the cluster. 
```

Comment:
Since `xpack.monitoring.elasticsearch.collection.enabled` defaults to `true`, should we ask the user to set it or would it be sufficient to ask them to verify that it's `true`, like we do above for the `xpack.monitoring.enabled` setting?

Reply:
Done!